### PR TITLE
feat(old-electron-app-notification): Implement

### PR DESCRIPTION
**Pull request by hristoterezov**
Detects if Jitsi Meet is running in old jitsi-meet-electron app and
warns the user for scurity issues.

### CODE REVIEW

File: react/features/old-client-notification/middleware.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * Notifies the feature that the action {@link APP_WILL_MOUNT} has being dispatched.
+ *
+ * @param {Store} store - The redux store in which the specified {@code action} is being dispatched.
+ * @param {Dispatch} next - The redux {@code dispatch} function to dispatch the specified {@code action}.
+ * @param {Action} action - The redux action {@code APP_WILL_MOUNT} which is being dispatched.
+ * @private
+ * @returns {Object} The new state that is the result of the reduction of the specified {@code action}.
+ */
+function _appWillMount(store, next, action) {
+    if (typeof interfaceConfig !== 'undefined'
```
Comment (saghul) [inline react/features/old-client-notification/middleware.js:33]: Please remove the interfaceConfig check, everybody running the app should get this.

File: react/features/old-client-notification/functions.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// @flow
+
+/**
+ * Returns true if Jitsi Meet is running in too old jitsi-meet-electron app and false otherwise.
+ *
+ * @returns {boolean} - True if Jitsi Meet is running in too old jitsi-meet-electron app and false otherwise.
+ */
+export function isOldJitsiMeetElectronApp() {
+    const match = navigator.userAgent.match(/(JitsiMeet)\s*\/\s*((\d+)\.[^\s]*)/);
```
Comment (saghul) [inline react/features/old-client-notification/functions.js:9]: Isn't there a specific Electron global we could use to confirm it's Electron? Just to be double-safe.

File: react/features/old-client-notification/middleware.js (hunk 1 above)
Comment (hristoterezov) [reply react/features/old-client-notification/middleware.js]: removed

File: react/features/old-client-notification/functions.js (hunk 2 above)
Comment (hristoterezov) [reply react/features/old-client-notification/functions.js:15]: used the browser check that we have

**Pull request merged**